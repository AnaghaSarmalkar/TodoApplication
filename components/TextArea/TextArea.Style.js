import { StyleSheet } from "react-native";

export const style = StyleSheet.create({
  container: {
    flex: 1,
  },
  description: {
    fontSize: 16,
  },
});
