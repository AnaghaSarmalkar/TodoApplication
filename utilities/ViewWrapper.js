import React from "react";
import { View } from "react-native";

export default function ViewWrapper(props) {
  return (
    <View
      {...props}
      // style={{
      //   flex: 1,
      //   alignItems: "center",
      //   justifyContent: "center",
      //   // paddingTop: insets.top,
      // }}
    >
      {props.children}
    </View>
  );
}
