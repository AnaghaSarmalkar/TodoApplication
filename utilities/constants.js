import { Platform } from "react-native";

export const isIos = Platform.OS === "ios";
export const isWeb = Platform.OS === "web";
export const isAndroid = Platform.OS === "android";
