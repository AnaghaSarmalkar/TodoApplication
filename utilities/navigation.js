export function setPageTitle(navigation, title) {
  if (title) {
    navigation.setOptions({
      title: title,
    });
  }
}
