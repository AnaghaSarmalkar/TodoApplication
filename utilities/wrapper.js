import React from "react";
import { SafeAreaView } from "react-native-safe-area-context";

function SafeAreaWrapper({ children }) {
  <SafeAreaView>{children}</SafeAreaView>;
}
