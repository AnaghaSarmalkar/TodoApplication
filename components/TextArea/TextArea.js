import React from "react";
import { Text } from "react-native";
import { style } from "./TextArea.Style";

export default function TextArea({ description }) {
  return <Text style={[style.description]}>{description}</Text>;
}
