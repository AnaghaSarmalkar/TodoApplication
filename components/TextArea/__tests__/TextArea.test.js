import React from "react";
import { shallow } from "../../../utilities/testing";
import TextArea from "../TextArea";

describe("", () => {
  it("renders Loading component", async () => {
    const description = "Test Description";
    const wrapper = shallow(<TextArea description={description} />);

    expect(wrapper).toMatchSnapshot();
  });
});
