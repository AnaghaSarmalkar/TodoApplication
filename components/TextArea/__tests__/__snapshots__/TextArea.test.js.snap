// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` renders Loading component 1`] = `
<View>
  <Text
    style={
      [
        {
          "fontSize": 16,
        },
      ]
    }
  >
    Test Description
  </Text>
</View>
`;
